src/focus-visible.js

```diff
@@ -25,15 +25,17 @@ function init() {
 
   /**
    * Helper function for legacy browsers and iframes which sometimes focus
-   * elements like document and body.
+   * elements like document, body, and non-interactive SVG.
    * @param {Element} el
    */
   function isValidFocusTarget(el) {
     if (
       el &&
       el !== document &&
       el.nodeName !== 'HTML' &&
-      el.nodeName !== 'BODY'
+      el.nodeName !== 'BODY' &&
+      el.classList &&
```

Comment:
can we do `'classList' in el && 'contains' in el.classList` to avoid breaking my brain?

Reply:
sure